applications/FluidDynamicsApplication/custom_elements/embedded_navier_stokes.h

```diff
@@ -477,25 +487,25 @@ class EmbeddedNavierStokes : public NavierStokes<TDim, TNumNodes>
                     N_aux_trans(i*BlockSize+comp, comp) = aux_N(i);
                 }
             }
-            
+
             // Contribution coming fron the shear stress operator
             auxLeftHandSideMatrix += weight*prod(N_aux_trans, aux_matrix_ACB);
-            
-            
+
```

Comment:
use noalias.

also, why didn t u make use of the trans(operator)?

Reply:
Because I directly construct the transpose matrix, so there is no original matrix to be transposed.